pyfar/io/io.py

```diff
@@ -230,12 +230,18 @@ def read(filename):
                         'different versions of Pyfar.')
                 collection[name] = obj
 
+        if 'builtin_wrapper' in collection:
+            for key, value in collection['builtin_wrapper'].items():
+                collection[key] = value
+            collection.pop('builtin_wrapper')
+
     return collection
 
 
 def write(filename, compress=False, **objs):
     """
-    Write any compatible pyfar object or numpy array as .far file to disk.
+    Write any compatible pyfar object or numpy array and often used builtin
+    types as .far file to disk.
```

Comment:
Are there any pyfar objects left, that we do not support? I would simplify this to:

```suggestion
    Write pyfar objects, numpy arrays, and Python built in types as .far file to disk.
```

Reply:
Well not all builtin types are actually supported, but those which are often used. But You're right with the pyfar objects. They are all supported at moment.